MCC-Axolotls/my-little-tree-backend
# Project Set Up - Created basic folder structure

This PR covers the Express set up for our backend project. The plan is to use a clean architecture with mongoose, I also installed bcrypt and jwt in case we want to manage auth token and encrypt the user's passwords.

<img width="657" alt="image" src="https://github.com/MCC-Axolotls/my-little-tree-backend/assets/9260288/dbe993bf-4810-4787-8c71-b5544327a528">

Browser Response:
<img width="719" alt="image" src="https://github.com/MCC-Axolotls/my-little-tree-backend/assets/9260288/1ac4be49-40ee-468b-82ef-46faac24fd6b">


In order to test the project follow the next steps:
1. Clone de repository
2. Make sure you have installed node 20 or latest (I recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your node versions)
3. Go to the root directoy
4. Run the command `npm install`
5. Once it's completed, run the command `npm run dev`
6. The server is going to start on http://localhost:8080
7. Use your broswer and navigate to http://localhost:8080/users
8. Verify the response is the following -> `{"success":true,"data":{"message":"Hello World"}}`
